Add First Steps topic (#7512)
DevExpress/devextreme-documentation: concepts/Common/First Steps/00 First Steps.md

---
##### Angular

> [important] You should be familiar with basic concepts and patterns of {Framework} to use DevExtreme components. If you are not, refer to the <a href="https://angular.dev/overview" target="_blank">{Framework} documentation</a> for a getting-started tutorial first.

Reviewer: ```suggestion
[important] You should be familiar with basic concepts and patterns of {Framework} to use DevExtreme components. If you are not, refer to the <a href="https://angular.dev/overview" target="_blank">{Framework} documentation</a> to follow its getting-started tutorial first.
```

##### Vue

> [important] You should be familiar with basic concepts and patterns of {Framework} to use DevExtreme components. If you are not, refer to the <a href="https://vuejs.org/guide/introduction.html#getting-started" target="_blank">{Framework} documentation</a> for a getting-started tutorial first.

Reviewer: ```suggestion
[important] You should be familiar with basic concepts and patterns of {Framework} to use DevExtreme components. If you are not, refer to the <a href="https://vuejs.org/guide/introduction.html#getting-started" target="_blank">{Framework} documentation</a> to follow its getting-started tutorial first.
```

##### React

> [important] You should be familiar with basic concepts and patterns of {Framework} to use this DevExtreme components. If you are not, refer to the <a href="https://react.dev/learn" target="_blank">{Framework} documentation</a> for a getting-started tutorial first.

Reviewer: ```suggestion
[important] You should be familiar with basic concepts and patterns of {Framework} to use DevExtreme components. If you are not, refer to the <a href="https://react.dev/learn" target="_blank">{Framework} documentation</a> to follow its getting-started tutorial first.
```

##### React

[important] You should be familiar with basic concepts and patterns of {Framework} to use this DevExtreme components. If you are not, refer to the <a href="https://react.dev/learn" target="_blank">{Framework} documentation</a> for a getting-started tutorial first.

---

In this step-by-step tutorial, you will build a simple task list application with a progress bar. By the end, you will have the following application:

<div class="simulator-desktop-container" data-view="/Content/Applications/25_1/FirstSteps/index.html, /Content/Applications/25_1/FirstSteps/index.js, /Content/Applications/25_1/FirstSteps/index.css"></div>

The application includes two DevExtreme components: [DataGrid](/Documentation/Guide/UI_Components/DataGrid/Overview/) and [ProgressBar](/Documentation/Guide/UI_Components/ProgressBar/Overview/). The DataGrid displays tasks, and you can mark them as complete or incomplete. The ProgressBar above the DataGrid indicates task completion progress.

> [note] This article explains how to build an application with DevExtreme components from scratch. For in-depth details, guides, API lists, and other tutorials, explore this website.

Reviewer 2: I'm not sure what I see the connection between the sentences. The first sentence says that I'm building something "from scratch". So I expect the second sentence to maybe say something like "if you are NOT doing it from scratch, then ...".

It seems like the end goal is to say "you can read this documentation to learn about our products"... which goes without saying. So maybe we don't need this note? 

```suggestion
```
